ci-operator/step-registry/aro-hcp/kql-verify/aro-hcp-kql-verify-commands.sh

```
@@ -0,0 +1,24 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# Start Kusto emulator via podman
+podman run -e ACCEPT_EULA=Y -m 4G -d -p 8080:8080 \
+  --name kusto-emulator \
+  mcr.microsoft.com/azuredataexplorer/kustainer-linux:latest
+
+# Wait for readiness
+for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)